hawc/apps/vocab/templates/vocab/toxref_browse.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'crumbless.html' %}
+
+{% block content %}
+  <a class="pull-right btn btn-primary" href="{% url 'vocab:api:toxref-nested' %}?format=xlsx">Download</a>
+  <h2>ToxRef Database Vocabulary</h2>
+  <div>
+    <p>
+      Experimental data extracted into HAWC is designed to enable filtering, sorting, and
```

Comment:
I left this template in case we're changing the text for the toxref vocab: if the text is consistent for all vocabularies, further refactoring is possible